go-controller/hybrid-overlay/pkg/controller/master.go

```diff
@@ -130,31 +131,32 @@ func NewMaster(kube kube.Interface,
 }
 
 // Run starts the controller
-func (m *MasterController) Run(stopCh <-chan struct{}) {
+func (m *MasterController) Run(stopCh <-chan struct{}, done *sync.WaitGroup) {
 	defer utilruntime.HandleCrash()
 	klog.Info("Starting Hybrid Overlay Master Controller")
 
-	klog.Info("Starting workers")
+	done.Add(3)
```

Comment:
nit. I would `done.Add(1)` before each goroutine instead because if someone copies/pastes the goroutine code to add a new informer, it's likely that they'll miss incrementing this here. Otherwise it probably warrants a comment.